crates/key-server/src/valid_ptb.rs

```diff
@@ -167,39 +194,41 @@ mod tests {
         let ptb = builder.finish();
         assert_eq!(
             ValidPtb::try_from(ptb).err(),
-            Some(InternalError::InvalidPTB)
+            Some(InternalError::InvalidPTB(
+                "Empty PTB input or command".to_string()
+            ))
         );
     }
 
     #[test]
     fn test_invalid_non_move_call() {
         let mut builder = ProgrammableTransactionBuilder::new();
         let sender = SuiAddress::random_for_testing_only();
-        let caller = builder.pure(sender).unwrap();
         let id = vec![1u8, 2, 3, 4];
         let id_caller = builder.pure(id.clone()).unwrap();
         let pkgid = ObjectID::random();
+
         builder.programmable_move_call(
             pkgid,
             Identifier::new("bla").unwrap(),
             Identifier::new("seal_approve_x").unwrap(),
             vec![],
-            vec![caller, id_caller],
```

Comment:
fixed a few tests here bc they would fail with invalid bcs for first arg, instead of what the test is meant to test for